groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -4335,7 +4387,8 @@ template <class TYPE>
 BSLS_KEYWORD_CONSTEXPR bool operator>(
                              const bsl::nullopt_t&,
                              const bsl::optional<TYPE>&) BSLS_KEYWORD_NOEXCEPT;
-    // Return 'false'.
+    // Return 'false'.  Note that 'bsl::nullopt_t' never orders before a
```

Comment:
```suggestion
    // Return 'false'.  Note that 'bsl::nullopt_t' never orders after a
```
?

Reply:
indeed. This is a copy paste error from NullableValue. I'll change it in my version of NullableValue, should we bother doing something in the current version ?

Reply:
I think as long as it will eventually end up fixed after all optional/NullableValue work is done, there's no need to do anything else.